clients/cli/src/main/java/org/apache/gravitino/cli/FullName.java

```diff
@@ -215,7 +215,7 @@ public boolean hasNamePart(int partNo) {
       int length = names.length;
       int position = partNo;
 
-      return position <= length;
+      return position >= 0 && position <= length;
```

Comment:
While I mentioned a negative value in the issue, we could change this to position> 0, as 0 is not a valid option either.